app/pages/project/project-page.spec.js

```diff
@@ -2,27 +2,59 @@ import React from 'react';
 import { mount, shallow } from 'enzyme';
 import { expect } from 'chai';
 import { project, workflow } from '../dev-classifier/mock-data';
-import ProjectPage from './project-navbar';
+import ProjectPage from './project-page';
+
+function Page() {
+  return (
+    <p>Hello world!</p>
+  );
+}
 
 describe('ProjectPage', () => {
   const background = {
     src: 'the project background image url'
   };
 
   it('should render without crashing', () => {
-    shallow(<ProjectPage />);
+    shallow(<ProjectPage><Page /></ProjectPage>);
   });
 
   it('should render the project nav bar', () => {
-    const wrapper = shallow(<ProjectPage />);
+    const wrapper = shallow(<ProjectPage><Page /></ProjectPage>);
     const navElement = wrapper.find('ProjectNavbar').first();
     expect(navElement).to.have.lengthOf(1);
   });
 
   it('should receive a background, project and workflow props', () => {
```

Comment:
In hindsight I should remove this test. It checks something that is already React's job, i.e. checking components receive props and I don't think we should test React.